### PR TITLE
cider-repl-result-prefix only inserted before the first result

### PR DESCRIPTION
**Pull request by lewang**
- subsequent chunks of result should not have this re-inserted.

See also https://github.com/clojure-emacs/cider/issues/315

-----------------

Before submitting the PR make sure the following things have been done (and denote this
by checking the relevant checkboxes):

- [ ] The commits are consistent with our [contribution guidelines][1]
- [ ] You've added tests (if possible) to cover your change(s)
- [ ] All tests are passing (`make test`)
- [ ] The new code is not generating bytecode or `M-x checkdoc` warnings
- [ ] You've updated the changelog (if adding/changing user-visible functionality)
- [ ] You've updated the readme (if adding/changing user-visible functionality)
- [ ] You've updated the refcard (if you made changes to the commands listed there)

Thanks!

*If you're just starting out to hack on CIDER you might find this [section of its
manual][2] extremely useful.*

[1]: https://github.com/clojure-emacs/cider/blob/master/.github/CONTRIBUTING.md
[2]: https://cider.readthedocs.io/en/latest/hacking_on_cider/

### CODE REVIEW

**Comment by dpsutton** [comment]
checks failing aren't your fault. They are from [a undefined variable](https://github.com/clojure-emacs/cider/blob/master/cider-stacktrace.el#L735).

**Comment by lewang** [comment]
thanks for the heads up.

**Comment by bbatsov** [comment]
@dpsutton Hmm, seems our build miraculously started working again. This sneaked in, while it wasn't working properly. I've just fixed it.

**Comment by bbatsov** [comment]
Now we started getting some odd errors with Emacs 26 when trying to contact the package.el repos - likely something changed there and TLS is now the default. Any ideas?

File: cider-repl.el [hunk 1]
```diff
@@ -698,7 +698,7 @@ If BOL is non-nil, emit at the beginning of the line."
           (cider-repl--insert-prompt cider-buffer-ns))))
     (cider-repl--show-maximum-output)))
 
-(defun cider-repl-emit-result (buffer string &optional bol)
+(defun cider-repl-emit-result (buffer string &optional bol show-prefix)
```
Comment (bbatsov) [inline cider-repl.el]: Likely this should move before the `bol` param, otherwise it looks kind of weird.

File: cider-interaction.el [hunk 2]
```diff
@@ -649,8 +649,10 @@ in the buffer."
 (defun cider-stdin-handler (&optional buffer)
   "Make a stdin response handler for BUFFER."
   (nrepl-make-response-handler (or buffer (current-buffer))
-                               (lambda (buffer value)
-                                 (cider-repl-emit-result buffer value t))
+                               (let (after-first-call)
```
Comment (bbatsov) [inline cider-interaction.el]: I'd add here a small comment explaining the need for this and maybe use the name `after-first-result-chunk`.

**Comment by bbatsov** [comment]
The change looks reasonable, but it needs a changelog entry.

**Comment by bbatsov** [comment]
@lewang Ping :-)

**Comment by bbatsov** [comment]
I guess I'll have to do the changes I mentioned myself. :-)

**Comment by lewang** [comment]
thanks @bbatsov

**Pull request merged**